### DISCUSSION TITLE
smartEQ3 not in the VST2 list anymore?

### DISCUSSION THREAD

**Discussion by mynet1982** [apohl79/audiogridder#1138] [Help]
Some versions ago I could use smartEQ3. Now, from Beta14 n 15 (maybe even earlier) It doesn't get approved by the plugin scanning process? It's not in the VST2 list anymore?
Tried many things but can't get it to work?
Someone know how to manually add it?

Thanks!

FL Studio
Win 10

**Comment by apohl79** [comment]
Did the scan fail or did it not scan?

**Reply by mynet1982** [reply]
Scan didn’t fail.
It’s just not in the list?
Is there a vst blacklist to change manually?
Thanks!

**Reply by apohl79** [reply]
All blacklisted plugins are at the bottom of the plugin list in the plugin manager. You can unblacklist it there. You can check the server log for the plugin to be scanned. If it is not scanned, it might under a location AG is not scanning.
